components/board/pi/common/common.go

```diff
@@ -17,12 +17,13 @@ var ModelName = resource.NewDefaultModel("pi")
 
 // ServoConfig is the config for a pi servo.
 type ServoConfig struct {
-	Pin       string   `json:"pin"`
-	Min       int      `json:"min,omitempty"`
-	Max       int      `json:"max,omitempty"`
-	StartPos  *float64 `json:"starting_position_degs,omitempty"`
-	HoldPos   *bool    `json:"hold_position,omitempty"` // defaults True. False holds for 500 ms then disables servo
-	BoardName string   `json:"board"`
+	Pin         string   `json:"pin"`
+	Min         int      `json:"min,omitempty"`
+	Max         int      `json:"max,omitempty"`
+	StartPos    *float64 `json:"starting_position_degs,omitempty"`
+	HoldPos     *bool    `json:"hold_position,omitempty"` // defaults True. False holds for 500 ms then disables servo
+	BoardName   string   `json:"board"`
+	MaxRotation int      `json:"max_rotation,omitempty"`
```

Comment:
Is there a good place to let people know that the config has been updated?

Reply:
I believe the docs team should be notified. Pinging @stevebriskin because I've forgotten the exact process, I believe we notify Matt Vella?

Reply:
@npentrel @mcvella